src/iap/modules/dev/components/dev-page/dev-page.component.html

```diff
@@ -0,0 +1,10 @@
+<div class="page">
```

Comment:
:point_right: :point_left: 
```suggestion
<div class="iap-page">
```